src/interface/src/app/login/login.component.html

```diff
@@ -51,6 +51,11 @@ <h1 class="signin-title">Sign in to Planscape</h1>
         </div>
         <mat-error class="mat-error" *ngIf="accountError">
           {{ accountError }}
+          <div *ngIf="offerReverify" class="post-error-option">
+            <a clickable [routerLink]="[]" (click)="resendVerification()"
```

Comment:
What does `clickable`  do?

Reply:
Ooops -- nothing in this case, except serve as a reminder that I need to remove meaningless directives when I change my strategy :man_facepalming: